OpenTabletDriver/Attributes/RangeSettingAttribute.cs

```diff
@@ -10,15 +10,15 @@ namespace OpenTabletDriver.Attributes
     [PublicAPI]
     public class RangeSettingAttribute : SettingAttribute
     {
-        public RangeSettingAttribute(string displayName, float min, float max, float defaultValue = 0f) : base(displayName)
+        public RangeSettingAttribute(string displayName, float min, float max, float step) : base(displayName)
         {
             Min = min;
             Max = max;
-            DefaultValue = defaultValue;
```

Comment:
unless `DefaultValue` is available somewhere else I don't think this should be dropped

Reply:
Available somewhere else.